README.md

```diff
@@ -45,9 +45,22 @@ It includes the following standards/guidelines:
 
 <p align="center"><strong>DO NOT USE ANY OF THE VERSIONS BELOW FOR NEW DOCUMENTS.</strong></p>
 
+### VPAT&reg; Version 2.5 (November 2023)
+
+VPAT version 2.5 has been updated to incorporate the [Web Content Accessibility Guidelines 2.2](https://www.w3.org/TR/WCAG22/) (WCAG 2.2) published in October 2023.
+
+It includes the following standards/guidelines:
+
+* [Web Content Accessibility Guidelines 2.0](https://www.w3.org/TR/2008/REC-WCAG20-20081211/)
+* [Web Content Accessibility Guidelines 2.1](https://www.w3.org/TR/WCAG21/)
+* [Web Content Accessibility Guidelines 2.2](https://www.w3.org/TR/WCAG22/)
+* [Revised Section 508 standards published January 18, 2017 and corrected January 22, 2018](https://www.access-board.gov/ict/)
+* [EN 301 549 Accessibility requirements for ICT products and services - V3.1.1 (2019-11)](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.01.01_60/en_301549v030101p.pdf)
+* [EN 301 549 Accessibility requirements for ICT products and services - V3.2.1 (2021-03)](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf)
+
 ### VPAT&reg; Version 2.4 Rev (March 2022)
```

Comment:
Similarly, I think we can just remove the update notes for the older versions. Maybe keep a cut-off date of 5 years, so we keep anything newer than 5 years old, and let the older versions drop off? 

(keep v1.0 though, since BMC and Springer Open are using it for now - I'd like to remove both of those documents but need to discuss first)